package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "start": "node .",
     "stop": "pkill levweb",
-    "dev": "NODE_ENV=development nodemon .",
+    "dev": "npm run mock-kc-proxy & NODE_ENV=development nodemon .",
```

Comment:
There is a small problem here in that the mock-kc-proxy will not be killed when the app is killed.

Might be nice to have a "stop:mock-kc-proxy" too.